mergin/merginproject.py

```diff
@@ -139,7 +139,11 @@ def project_full_name(self) -> str:
         """Returns fully qualified project name: <workspace>/<name>"""
         if self._metadata is None:
             self._read_metadata()
-        return self._metadata["name"]
+        return (
+            self._metadata["name"]
+            if "/" in self._metadata["name"]
+            else f"{self._metadata['namespace']}/{self._metadata['name']}"
+        )
```

Comment:
My preference would be that we detect in _read_metadata() whether we deal with new or old metadata, and then the metadata access methods could just use this kind of flag (e.g. `if self._old_metadata_format`) to make the code easier to understand

Reply:
Changed to use flag